sonar-bukkit/src/main/java/xyz/jonesdev/sonar/bukkit/SonarBukkit.java

```diff
@@ -103,8 +104,10 @@ public void enable() {
     // Register Sonar command
     Objects.requireNonNull(getPlugin().getCommand("sonar")).setExecutor(new BukkitSonarCommand());
 
-    // Make sure to inject into the server's connection handler
-    getPlugin().getServer().getScheduler().runTask(getPlugin(), FallbackBukkitInjector::inject);
+    if (SonarBukkitPlugin.lateBind) {
+      //Bukkit.getScheduler().runTask(getPlugin(), FallbackBukkitInjector::inject);
+      Bukkit.getScheduler().runTask(getPlugin(), FallbackBukkitInjector::inject);
```

Comment:
Why is there a commented line here?
Why not use `getPlugin().getServer()` instead of `Bukkit` to remove one import (codestyle)?

Reply:
Solved